firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -221,7 +221,11 @@ class BrowserViewController: UIViewController,
     }
 
     private let bottomBlurView: UIVisualEffectView = .build { view in
-        view.effect = UIBlurEffect(style: .systemUltraThinMaterial)
+        view.effect = if #available(iOS 26.0, *) {
+            UIGlassEffect(style: .regular)
+        } else {
+            UIBlurEffect(style: .systemUltraThinMaterial)
+        }
```

Comment:
@PARAIPAN9 this will fail on BR. The way we have been doing this is using conditional compilation any time we have iOS 26 specific APIs.
```suggestion
#if canImport(FoundationModels)
    view.effect = if #available(iOS 26.0, *) {
        UIGlassEffect(style: .regular)
    } else {
        UIBlurEffect(style: .systemUltraThinMaterial)
    }
#else
    view.effect = UIBlurEffect(style: .systemUltraThinMaterial)
#endif
```